Editor/SimpleGraphQL/GraphQLImporterV1.cs

```diff
@@ -16,30 +17,37 @@ public override void OnImportAsset(AssetImportContext ctx)
         {
             var lexer = new Lexer();
             var parser = new Parser(lexer);
+            var regex = new Regex("#import \\\"(?<path>[\\w\\/.]+)\\\"");
```

Comment:
I usually like to leave a comment explaining what a regex is supposed to do when I use them because they can increase parsing time if you're trying to understand what a function does without a description.

Or put them behind a named global (module attribute in elixir for example)